### ISSUE TITLE
[Accessibility][PDF Viewer] Icon buttons lack accessible names

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1749]
**Steps to Reproduce**
1. Enable NVDA screen reader.
2. Open page in Google Chrome.
3. Press Tab to move focus to the Search (magnifying glass icon) button and/or 3 dot icon menu buttons

**Current Behavior**
Search buttons and 3 dots icon menu buttons have no accessible name. 
The search menu button is announced simply as "button" and the 3 dots icon menu button is announced as "menu button" (the aria-haspopup attribute on this button causes NVDA to identify it as a menu button). Screen reader users are not informed of the purpose of the buttons, and users of voice input cannot target the buttons directly.

**Expected behavior/code**
All controls provide descriptive accessible names.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Assistive technology users

**WCAG SC**
4.1.2 Name, Role, Value

**Possible Solution**
For icon buttons, accessible names can be provided with an aria-label attribute or with visually-hidden text content within the button.

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by glasert** [comment]
This issue won't be fixed as the acrobat team is working on a modern viewer which addresses accessibility and will replace the current viewer in 2022. Therefore we defer and close this issue
